components/kyma-environment-broker/internal/broker/config.go

```diff
@@ -0,0 +1,6 @@
+package broker
+
+type KEBConfig struct {
```

Comment:
I feel the purpose of this new config is a bit confusing. How about sending down the whole `cmd/broker.Config` struct from `main()` to the orchestration managers, and would just use that.

Reply:
I previously moved the whole `cmd/broker.Config` under /internal/broker and it caused a import cycle. Standalone KEBConfig was my solution to avoid that. I will try to move the whole `cmd/broker.Config` under orchestration/manager